frontend/src/lib/util/tooltip.ts

```diff
@@ -1,31 +1,43 @@
 import { tooltipState } from '$lib/stores';
 
 export function tooltip(node: HTMLElement, params: { text: string }) {
-	function handleFocus(e: MouseEvent) {
-		tooltipState.set({ hover: true, mousePos: { x: e.clientX, y: e.clientY }, text: params.text });
+	let isHovering = false;
```

Comment:
Why this change? Before we were not adding listeners that are not needed. Now we call the move function even if we're not hovering. Seems unnecessary.

Reply:
Ah good point. Changed it back but still in a way that the tooltip text updates.